src/v2/Components/Inquiry/Hooks/useInquiryContext.tsx

```diff
@@ -99,28 +106,32 @@ const InquiryProvider: React.FC<InquiryProviderProps> = ({
    * this data to re-render any views; it's only used to make decisions between views.
    */
   const context = useRef<Context>({
+    ...DEFAULT_CONTEXT,
     askSpecialist: !!askSpecialist,
-    collectorLevel: me?.collectorLevel,
-    isLoggedIn: !!me,
-    location: !!me?.location?.city ? { city: me.location.city } : null,
-    phone: me?.phone,
-    profession: me?.profession,
-    shareFollows: !!me?.shareFollows,
   })
 
   // Mutate the decision context directly
-  const setContext = (updatedContext: Partial<Context>) => {
+  const setContext = useCallback((updatedContext: Partial<Context>) => {
     context.current = { ...context.current, ...updatedContext }
     return context
-  }
+  }, [])
 
   const [inquiry, setInquiry] = useState<InquiryState>({
     message: DEFAULT_MESSAGE,
   })
 
+  const handleClose = () => {
+    if (context.current.requiresReload) {
+      window.location.reload()
```

Comment:
Def bringing back code ptsd from when we launched new app shell - can we add a comment here about why this is needed?

Reply:
Yeah, good idea.